### ISSUE TITLE
Avoid App Store updates & update prompts

### ISSUE THREAD

**Issue by calimarkus** [RobotsAndPencils/XcodesApp#209]
**Tell us how we can improve Xcodes**
Currently I have 13.3.1 installed through `Xcodes.app`, but the AppStore now shows me a pending update for 13.3.1 - the same exact version? How can that be avoided? Could `Xcodes.app` help disable the AppStore installation somehow? (Some config/plist/defaults setting maybe?)

**Is your feature request related to a problem? Please describe.**
It's a feature request.

**What would you like to see? How would you like it to work?**
I would like the App Store to ignore Xcode or alternatively to at least recognize, that the latest version is already installed.

**Comment by MattKiazyk** [comment]
Hi @calimarkus a few people have asked how, and I still haven't figured it out and my searches have come up empty handed.

I'm curious if the new symlink preference (create an Xcode.app link to the current version) would help?

**Comment by calimarkus** [comment]
Sadly the symlink didn't help so far (at least not by creating it after the fact).

I also tried some manual renaming initially. So not sure if there's anything to the specific timing of things in my case. I wonder if fully uninstalling the AppStore version, before installing through Xcodes solves it. But testing that out on my old MacBook Air will take sooo long 😅

**Comment by MattKiazyk** [comment]
@calimarkus Came across this tweet: https://twitter.com/genegoykhman/status/1519833821964820483?s=20&t=x6XVojBGy5-805lRaXzPOg perhaps that works?

**Comment by calimarkus** [comment]
I think I got it finally resolved on my personal machine. Here's what I did:

I was in a pretty strange state, where the AppStore claimed that I have a pending Xcode update for 13.3.1. Even though I had 13.4 installed already (through Xcodes). The Xcode product page itself showed the "GET" button though, as if I didn't have Xcode installed..:

Product page | Updates page
-- | --
<img width="150" alt="Screen Shot 2022-05-18 at 11 24 12 AM" src="https://user-images.githubusercontent.com/807039/168946398-3e2ee710-9c39-4fbf-8bcb-33ac606f2ac0.png"> | <img width="300" alt="Screen Shot 2022-05-18 at 11 24 22 AM" src="https://user-images.githubusercontent.com/807039/168946405-4050cabf-c98e-45a5-a76e-dc6487417917.png">

So I figured some internal AppStore state is borked. I followed this article https://macpaw.com/how-to/clear-app-store-cache to find the caches of the AppStore. I ended up deleting both the following paths fully:

```
$TMPDIR../C/com.apple.appstore/
~/Library/Caches/com.apple.appstore/
```

I then also logged out of the AppStore & restarted my Mac. And finally... no pending Xcode update is shown! Yay.
